test/data/hover/test_role/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Is this file required to be part of fixtures folder?

Reply:
No. Its not required. Actually that was the outcome of `ansible-galaxy init` command.
I will remove them.